ee/tabby-webserver/graphql/schema.graphql

```diff
@@ -1,10 +1,40 @@
+type RegisterResponse {
+  accessToken: String!
+  refreshToken: String!
+  errors: [AuthError!]!
```

Comment:
Should we utilize juniper's error handling directly?

https://graphql-rust.github.io/juniper/master/types/objects/error_handling.html#error-payloads-null-and-partial-errors

it generates `errors` field automatically